api/src/controllers/nfl-teams.controller.ts

```diff
@@ -0,0 +1,33 @@
+import { NextFunction, Request, Response } from 'express';
+import { Container } from 'typedi';
+import { CreateNFLTeamDto } from '@dtos/nfl-teams.dto';
+import { NFLTeam } from '@interfaces/nfl-teams.interface';
+import { NFLTeamService } from '@services/nfl-teams.service';
+
+
+export class NFLTeamController {
+  public nflteam = Container.get(NFLTeamService);
+
+  public getNFLTeams = async (req: Request, res: Response, next: NextFunction) => {
+    try {
+      const findAllNFLTeamsData: NFLTeam[] = await this.nflteam.findAllNFLTeam();
+
+      res.status(200).json({ data: findAllNFLTeamsData, message: 'findAll' });
+    } catch (error) {
+      next(error);
+    }
+  };
+
+  public getNFLTeamById = async (req: Request, res: Response, next: NextFunction) => {
```

Comment:
These look great. You might be able to add return types I can't remember though because I these aren't traditional typescript functions.